project/api/models.py

```diff
@@ -297,25 +297,6 @@ def clean(self, *args, **kwargs):  # pylint: disable=W0221
                     "Only Polygon and MultiPolygon objects are acceptable geometry types."
                 )
 
-            if (
-                SpacetimeVolume.objects.filter(
-                    start_date__lte=self.end_date,
-                    end_date__gte=self.start_date,
-                    territory__intersects=self.territory,
-                )
-                .exclude(pk=self.pk)
-                .exists()
-            ):
-                raise ValidationError(
-                    "Another SpaceTimeVolume overlaps this polygon: "
```

Comment:
Can we use a condition based on a _flag_ or an environment variable, to disable this check?

Reply:
Just to disable it until we will fix the database?